### ISSUE TITLE
Search bar not reachable with keyboard navigation

### ISSUE THREAD

**Issue by cassidyjames** [ubuntu/app-center#1595]
When opening App Center, I cannot seem to focus the search bar using the keyboard; tabbing and using the arrow keys just switches focus between items in the main view (e.g. app cards)

**Comment by maubuz** [comment]
Related to #1475

**Comment by dominiksr** [comment]
It's very frustrating.

**Comment by d-loose** [comment]
Fixed by #1574
